lib/foreman_maintain/concerns/downstream.rb

```diff
@@ -101,22 +122,42 @@ def puppet4_repo(full_version)
       end
 
       def common_repos(full_version)
-        repos_arrary = if ENV['FOREMAN_MAINTAIN_USE_BETA'] == '1'
-                         ["rhel-#{el_major_version}-server-satellite-maintenance-6-beta-rpms",
-                          "rhel-#{el_major_version}-server-satellite-tools-6-beta-rpms"]
-                       else
-                         ["rhel-#{el_major_version}-server-satellite-maintenance-6-rpms",
-                          "rhel-#{el_major_version}-server-satellite-tools-#{full_version}-rpms"]
-                       end
+        sat_maint_version = '6'
+        sat_client_version = full_version
+
+        if el7?
+          if use_beta_repos?
+            repos = ["rhel-#{el_major_version}-server-satellite-maintenance-#{sat_maint_version}-beta-rpms",
+                     "rhel-#{el_major_version}-server-satellite-tools-#{sat_client_version}-beta-rpms"]
+          else
+            repos = ["rhel-#{el_major_version}-server-satellite-maintenance-#{sat_maint_version}-rpms",
+                     "rhel-#{el_major_version}-server-satellite-tools-#{sat_client_version}-rpms"]
+          end
+        else
+          if use_beta_repos?
+            # FIXME: this is for sure wrong
+            repos = ["satellite-maintenance-#{sat_maint_version}-for-rhel-#{el_major_version}-x86_64-rpms",
+                     "satellite-client-#{sat_client_version}-for-rhel-#{el_major_version}-x86_64-rpms"]
+          else
+            repos = ["satellite-maintenance-#{sat_maint_version}-for-rhel-#{el_major_version}-x86_64-rpms",
+                     "satellite-client-#{sat_client_version}-for-rhel-#{el_major_version}-x86_64-rpms"]
+          end
+        end
 
-        return repos_arrary.first(1) if feature(:satellite)
 
-        repos_arrary
+        return repos.first(1) if feature(:satellite)
```

Comment:
why do we even have the client repos here? are they used on capsules (I thought not)

Reply:
Correct, I checked the old changes but cant see a reason or discussion on why it was added. Safe to remove!